### PR TITLE
Added the challenge objectives.

### PR DESCRIPTION
**Pull request by Estrax**

### CODE REVIEW

**Comment by Estrax** [comment]
Finished Project I

File: Project-I-Social-Card/social-card/src/components/CardComponents/Card.css [hunk 1]
```diff
@@ -1 +1,28 @@
 /* All styles for your Card Components live here */
+
+.img-card {
```
Comment (Estrax) [inline Project-I-Social-Card/social-card/src/components/CardComponents/Card.css:3]: missed the red background + border-radius 50-100% + white fill.

File: Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js [hunk 2]
```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+
+const ImageThumbnail = () => {
+    return (
+        <img
```
Comment (Estrax) [inline Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js:6]: or the container with the appropriate styles there and the img tag inside.

File: Project-I-Social-Card/social-card/src/App.css [hunk 3]
```diff
@@ -2,3 +2,7 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.container {
+  width: 500px;
```
Comment (BrandonVilaychith) [inline Project-I-Social-Card/social-card/src/App.css:7]: For some reason when I look at it, it isn't getting on constrained by the width.

**Review by BrandonVilaychith — COMMENTED** [review]
The project looks great overall and amazing work on the footer itself and the use of the moment.js. Amazing job on the git commits like always. On the calculator nice job on the use of mapping in your components to display them.

// Project Rating 
3

**Pull request merged**